src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,61 @@
+.stopwatch {
+  position: relative;
+  height: $main-size;
+  width: $main-size;
+
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  &__seconds {
+    position: absolute;
+    height: $second-height;
+    width: 1.5vmin;
+    top: 2vmin;
+    left: 50%;
+
+    transform: translateX(-50%);
+    transform-origin: bottom center;
+    animation: move 60s linear 0s infinite;
+    animation-fill-mode: backwards;
+
+    background-color: #2c8000;
+  }
+
+  &__minutes {
+    position: absolute;
+    height: $minute-height;
+    width: 3vmin;
+    top: 20vmin;
+    left: 50%;
+
+    transform: translateX(-50%);
+    transform-origin: bottom center;
+    z-index: -1;
```

Comment:
The 'z-index' property is set to -1, which will place the '.stopwatch__minutes' element behind the root of the stacking context. Make sure this is the intended behavior, as it may cause the minutes hand to be hidden behind other elements.